src/API/AbstractPluginActions.php

```diff
@@ -85,11 +85,19 @@ public function login()
         $isCreated = $this->dataStore->createUserDataStore($requestBody["apiKey"], $requestBody["email"], null, null);
```

Comment:
https://github.com/cloudflare/cloudflare-plugin-backend/blob/master/src/API/Client.php#L22-L23 sets the apikey/email based on dataStore values.  Have to create them some how or modify the api client.